fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraBinaryImpl.java

```diff
@@ -246,7 +248,7 @@ public URI getContentDigest() {
     public String getMimeType() {
         try {
             if (hasProperty(HAS_MIME_TYPE)) {
-                return getProperty(HAS_MIME_TYPE).getString();
+                return getProperty(HAS_MIME_TYPE).getString().replace(FIELD_DELIMITER + XSDstring.getURI(), "");
```

Comment:
So you are suggesting removing that construct altogether? What would you propose to replace that with?

To me, that seems like a separate issue.